### PR TITLE
Turn on phone number of HQ signup AB test

### PR DESCRIPTION
**Pull request by mkaczmarczyksoldevelo**

### CODE REVIEW

**Comment by orangejenny** [comment]
The data currently being sent with events should be attached to the user, not to the events themselves. You can use a call to `identifyTraits` for this.

File: corehq/apps/registration/static/registration/js/new_user.ko.js [hunk 1]
```diff
@@ -298,6 +301,12 @@ hqDefine('registration/js/new_user.ko', function () {
                     self.currentStep(_nextStep);
                 });
 
+            if (_nextStep === 1) {
```
[the remark this answers is not in the document]
Comment (mkaczmarczyksoldevelo) [reply corehq/apps/registration/static/registration/js/new_user.ko.js]: @orangejenny did that

**Comment by mkaczmarczyksoldevelo** [comment]
@orangejenny for `identifyTraits` I have added this to the second event because the first is sent when the user visits the register website before he/she have provided an email. In the second step, I've added the `identifyTraits`.

**Comment by orangejenny** [comment]
Hey, the updates look good, but I think calling `identifyTraits` is sufficient, I don't think the data also needs to be passed along to the events.

**Comment by mkaczmarczyksoldevelo** [comment]
@orangejenny fixed accordingly to our skype talk.

File: corehq/apps/registration/static/registration/js/new_user.ko.js [hunk 2]
```diff
@@ -38,11 +38,14 @@ hqDefine('registration/js/new_user.ko', function () {
 
     // Can't set up analytics until the values for the A/B tests are ready
     _kissmetrics.whenReadyAlways(function() {
-        _private.isAbPhoneNumber = _kissmetrics.getAbTest('New User Phone Number') === 'show_number';
+        _private.isAbPhoneNumberShown = _kissmetrics.getAbTest('New User Phone Number') === 'show_number';
+        _kissmetrics.track.event("Viewed CommCare signup page", {
```
Comment (orangejenny) [inline corehq/apps/registration/static/registration/js/new_user.ko.js]: This event doesn't need the data, either.

Comment (mkaczmarczyksoldevelo) [reply corehq/apps/registration/static/registration/js/new_user.ko.js]: ok, so `identifyTraits` it is

**Review by orangejenny — APPROVED** [review]

**Pull request merged**